src/pages/EmergencyViewPage/EmergencyViewPage.tsx

```diff
@@ -94,7 +94,7 @@ export const EmergencyViewPage: React.FC = () => {
         return (
           <div
             className="EmergencyView__weekdays__column"
-            key={day.toISOString()}
+            key={`${day.toISOString()}${dayIndex}`}
```

Comment:
```suggestion
            key={`${day.toISOString()}-${dayIndex}`}
```

Maybe possibly even `day.getTime()`?